### PR TITLE
Update to use the latests gfx 0.6.

### PR DESCRIPTION
**Pull request by csherratt**
This will probably need to be revisited. I don't think I did this
the best way possible, just the easiest way. Duplicating the DrawState
is pretty expensive space wise.

requires: https://github.com/gfx-rs/gfx-rs/pull/762

### CODE REVIEW

**Comment by kvark** [comment]
I'm not sure if `DrawState` cloning is really heavy though. It's a POD structure of less than 100 bytes, supposedly.

**Comment by kvark** [comment]
Could you publish the patched gfx-rs and restart Travis on this one?

File: src/phase/phase.rs [hunk 1]
```diff
@@ -5,6 +5,7 @@ use std::collections::HashMap;
 use draw_queue;
 use gfx;
 use mem;
+use draw_state::DrawState;
```
Comment (kvark) [inline src/phase/phase.rs]: I think gfx-rs re-exports it, so no need to depend on draw_state at all.

File: Cargo.toml [hunk 2]
```diff
@@ -29,7 +29,6 @@ git = "https://github.com/gfx-rs/draw_state"
 git = "https://github.com/gfx-rs/gfx-rs"
 
 [dev_dependencies.glutin]
-git = "https://github.com/tomaka/glutin"
```
Comment (kvark) [inline Cargo.toml]: How does this work?
I think dev_dependencies should also be switched to crates.io

**Comment by csherratt** [comment]
I will have to tweak this. I was using a `gfx-rs` overload and it looks like I missed some dependencies.

**Comment by kvark** [comment]
I finished the work in #36

**Pull request closed without merging**